main.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import FastAPI, Depends
+import models 
+import schemas
+from database import engine, SessionLocal
+from sqlalchemy.orm import Session
+
+models.Base.metadata.create_all(engine)
+app = FastAPI()
+
+def get_db():
+    db = SessionLocal()
+    
+    try:
+        yield db
+        
+    finally:
+        db.close()
+
+#Create API to get user ID and return user points and highest points and expertise label.
+@app.get('/userprofile/{id}', response_model = schemas.points_response)
+def points(id, db:Session = Depends(get_db)):
+    profile = db.query(models.user_profiles).filter(models.user_profiles.user_id == id).all()
+    if models.user_profiles.points == 0:
```

Comment:
You need to return data in json
`{"points": <points>, "highest_points": <highest points>,"expertise_label": <label>}`